utils.groovy

```diff
@@ -213,4 +213,8 @@ def getDockerBuildOs(String osName) {
   }
 }
 
+def getDockerTag() {
+  return "${env.IS_PRO ? 'pro-' : ''}${env.OS}-${env.ARCH}-${env.RSTUDIO_VERSION_FLOWER}"
```

Comment:
I think we still need to call `getDockerBuildOs` instead of passing `env.OS` directly

Reply:
Good catch!